src/components/atoms/Button/Button.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = styled.button`
+  margin: 15px 0;
+  padding: ${({ isBig }) => (isBig ? "10px 38px" : "7px 20px")};
+  font-size: ${({ isBig, theme: { fontSize } }) => (isBig ? fontSize.m : fontSize.s)};
+  background-color: ${({ theme }) => theme.colors.lightPurple};
+  border-radius: 20px;
+  border: none;
+  font-weight: bold;
+  color: ${({ theme }) => theme.colors.darkGrey};
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  position: relative;
+  top: 0;
+  left: 0;
+  cursor: pointer;
+`;
```

Comment:
kwestia struktury komponentów, dawniej korzystałem ze struktury atomic design, ale teraz widze że to było tragiczne

w folderze components trzymam jedynie komponenty które są reużywalne w wielu miejscach, i dziele sobie to na takie podfoldery:

```
components
    client (provider, layout, loader)
    errors (fetching-error, no-content, not-found)
    form (wszystkie elementy forma - button, input, select, checkbox itd...)
    ui (cała reszta komponentów które są reużywalne, a nie pasują do innych)
    modals (modale)
```

w przypadku kiedy buduje sobie strone główną powiedzmy nazywa się dashboard, to wszystkie komponenty trzymam wewnątrz tego folderu

załóżmy coś takiego - buduje dashboard, który zawiera header, banner, informacje o najnowszych mintach, jakiś komponent do wyświetlania nft, footer

Dashboard
  - Header
  - Banner
  - Latest mints
  - New nfts
  - Footer

więc tak:
**Header** - prawdopodobnie reużywalny, będzie się pojawiał na każdej możliwej stronie więc wrzucam go do folderu components, później mogę wykorzystać go np w `page.js` i zrobić komponent 
```
return (
  <Wrapper>
    <Header />
    {children}
  </Wrapper>
)
``` 
**Banner** - on będzie wyświetlany tylko i wyłącznie na tej stronie, więc wrzucam go do ścieżki:
`src/pages/dashboard/banner/banner.js`

**Latest mints** - też wyświetlany tylko na dashboardzie, ścieżka:
`src/pages/dashboard/latest-mints/latest-mints.js`

**New nfts** - znowu tylko w dashboardzie, ścieżka:
`src/pages/dashboard/new-nfts/new-nfts.js`

**Footer** - reużywalny, wrzucam do `components/ui` i wykorzystuje tak samo w komponencie `page.js` gdzie finalnie dostajemy coś takiego
```
return (
  <Wrapper>
    <Header />
    {children}
    <Footer />
  </Wrapper>
)
``` 

finalnie dostajemy taką strukturę:

```

dashboard
   banner
      banner.js
      banner.styles.js (jeżeli potrzebne)
      banner.constants.js (jeżeli potrzebne - jakieś stałe wartości itd)
      banner.types.ts (w przypadku używania typescripta)
   latest-mints
      latest-mints.js
      latest-mints.styles.js (jeżeli potrzebne)
      latest-mints.constants.js (jeżeli potrzebne - jakieś stałe wartości itd)
      latest-mints.types.ts (w przypadku używania typescripta)
   new-nfts
      new-nfts.js
      new-nfts.styles.js (jeżeli potrzebne)
      new-nfts.constants.js (jeżeli potrzebne - jakieś stałe wartości itd)
      new-nfts.types.ts (w przypadku używania typescripta)
   dashboard.js
   dashboard.styles.js (jeżeli potrzebne)
```